share/spack/gitlab/pr_pipeline.yml

```diff
@@ -17,4 +17,15 @@ merge_pipeline:
     SPACK_REF: develop
   trigger:
     project: spack/e4s
-    strategy: depend
\ No newline at end of file
+    strategy: depend
+
+release_pipeline:
+  only:
+  - releases/v0.15
+  variables:
+    SPACK_REPO: https://github.com/spack/spack.git
+    SPACK_REF: releases/v0.15
```

Comment:
If this branch is going potentially change while pipelines are queued, but not yet running (not as likely as on `develop`, I'll grant you), then we may want do what I've proposed in #17571 to make sure each pipeline gets triggered with a specific SHA of spack to test.